.eslintrc.js

```diff
@@ -67,6 +67,22 @@ module.exports = {
       'jsdoc/require-description': OFF
     }
   }, {
+    files: ['packages/@react-aria/focus/**/*.ts', 'packages/@react-aria/focus/**/*.tsx'],
+    rules: {
+      'no-restricted-globals': [
+        ERROR,
```

Comment:
Ensures that the entire @react-aria/focus package now uses `getOwnerDocument` instead of `document`